icons/scilab.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Scilab</title><path d="M20.4376 0H3.5624C1.5981 0 0 1.5981 0 3.5624v16.8752C0 22.4019 1.5981 24 3.5624 24h16.8752C22.4019 24 24 22.4019 24 20.4376V3.5624C24 1.5981 22.4019 0 20.4376 0zm2.5406 20.4376c0 1.4009-1.1397 2.5406-2.5406 2.5406H3.5624c-1.4009 0-2.5407-1.1397-2.5407-2.5406V3.5624c0-1.4009 1.1397-2.5407 2.5407-2.5407h16.8752c1.4009 0 2.5406 1.1397 2.5406 2.5407v16.8752zM20.1891 1.8031H3.8109c-1.0967 0-1.989.8923-1.989 1.989v16.4157c0 1.0967.8923 1.989 1.989 1.989h16.3783c1.0967 0 1.989-.8923 1.989-1.989V3.7921c-.0001-1.0967-.8923-1.989-1.9891-1.989zm-.8915 7.6763c.9954 0 1.8024.8172 1.8024 1.8252s-.807 1.8252-1.8024 1.8252-1.8024-.8172-1.8024-1.8252c0-1.0081.807-1.8252 1.8024-1.8252zm-1.8755-1.1206c0 1.008-.807 1.8252-1.8024 1.8252s-1.8024-.8172-1.8024-1.8252.807-1.8252 1.8024-1.8252c.9955 0 1.8024.8171 1.8024 1.8252zm-4.7892-5.3225c.9954 0 1.8024.8172 1.8024 1.8252s-.807 1.8252-1.8024 1.8252-1.8024-.8172-1.8024-1.8252.807-1.8252 1.8024-1.8252zm-8.3523 13.581c-.9954 0-1.8024-.8172-1.8024-1.8252s.807-1.8252 1.8024-1.8252 1.8024.8172 1.8024 1.8252c0 1.0081-.8069 1.8252-1.8024 1.8252zm3.5299 2.699c-.9954 0-1.8024-.8172-1.8024-1.8252s.807-1.8252 1.8024-1.8252 1.8024.8172 1.8024 1.8252-.807 1.8252-1.8024 1.8252zm.4636-8.8733c-.9954 0-1.8024-.8172-1.8024-1.8252s.807-1.8252 1.8024-1.8252 1.8024.8172 1.8024 1.8252-.807 1.8252-1.8024 1.8252zm3.9794 2.6335c-.9954 0-1.8024-.8172-1.8024-1.8252s.807-1.8252 1.8024-1.8252 1.8024.8172 1.8024 1.8252-.807 1.8252-1.8024 1.8252zm3.6413 2.5884c-.9954 0-1.8024-.8172-1.8024-1.8252s.807-1.8252 1.8024-1.8252 1.8024.8172 1.8024 1.8252c.0001 1.008-.8069 1.8252-1.8024 1.8252zm3.9047 1.8814c-.9954 0-1.8024-.8172-1.8024-1.8252s.807-1.8252 1.8024-1.8252 1.8024.8172 1.8024 1.8252-.807 1.8252-1.8024 1.8252z"/></svg>
```

Comment:
Good find, the app icon will look better!
I'm seeing small differences in 4 corners (yours in red):
![image](https://github.com/simple-icons/simple-icons/assets/12817388/cdc5fba1-3b9f-42fd-bdff-3b53523b19f9)

Maybe we'll want to ask another maintainers in case the problem is on my side 😅